### PR TITLE
WAI-990: Filter out remaining tupaia data from lesmis admin panel

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Add filter to permission groups endpoint in meditrak-server
- Remove demo land default from entities endpoint 
- Add custom filtering in lesmis to only get 'lesmis' platform users
- Remove viz builder buttons from lesmis admin panel
- Remove PermissionGroups and AccessRequests tables from lesmis
---

### CODE REVIEW

File: packages/lesmis/src/routes/AdminPanelRoutes.js [hunk 1]
```diff
@@ -128,22 +132,12 @@ export const ROUTES = [
...
-      },
-      {
-        label: 'Permission Groups',
-        to: '/permission-groups',
-        component: PermissionGroupsPage,
-      },
-      {
-        label: 'Access Requests',
-        to: '/access-requests',
-        component: AccessRequestsPage,
```
Comment (tcaiger) [inline packages/lesmis/src/routes/AdminPanelRoutes.js:146]: Remove access requests as it is not possible to make access requests on lesmis

File: packages/lesmis/src/routes/AdminPanelRoutes.js [hunk 2]
```diff
@@ -128,22 +132,12 @@ export const ROUTES = [
...
       },
       {
         label: 'Permissions',
         to: '/permissions',
-        component: PermissionsPage,
-      },
-      {
-        label: 'Permission Groups',
-        to: '/permission-groups',
-        component: PermissionGroupsPage,
```
Comment (tcaiger) [inline packages/lesmis/src/routes/AdminPanelRoutes.js:141]: remove permission groups as they are a defined list on lesmis and shouldn't be edited.

**Comment by tcaiger** [comment]
To whoever reviews this. I am merging this PR into the epic branch so that I can demo the epic branch with all the latest changes. Please still review it here and I can incorporate the suggestions into the [epic branch](https://github.com/beyondessential/tupaia/pull/3605) before they get merged to dev.

**Review by IgorNadj — COMMENTED** [review]
LGTM

**Pull request merged**